includes/core.php

```diff
@@ -21,6 +21,32 @@ function setup() {
 		return __NAMESPACE__ . "\\$function";
 	};
 
+	$errors   = array();
+	$warnings = array();
+
+	if ( ! defined( 'SOPHI_WP_VERSION' ) ) {
+		$warnings[] = __( 'Sophi Debug Bar requires Sophi for WordPress plugin', 'sophi-debug-bar' );
+	} elseif ( version_compare( SOPHI_WP_VERSION, '1.0.15', '<=' ) ) {
+		$warnings[] = __( 'Sophi Debug Bar requires Sophi for WordPress version 1.0.14 or higher', 'sophi-debug-bar' );
+	}
+
+	if ( ! class_exists( 'Debug_Bar' ) ) {
+		$errors[] = __( 'Sophi Debug Bar requires Debug Bar plugin', 'sophi-debug-bar' );
+	}
+
+	if ( ! empty( $warnings ) ) {
+		show_notices( $warnings );
+	}
+
+	if ( ! empty( $errors ) ) {
+		show_notices( $errors, 'error' );
+		// Stop plugin.
+		return;
+	}
+
+	// Early initialization of the panel before main debug bar plugin.
+	require_once SOPHI_DEBUG_BAR_PATH . '/includes/classes/panel.php';
```

Comment:
Some Sophi requests are triggered before the Debug Bar initialize their panels, so we setup the instance as soon as possible.